Examples/Detectors/DD4hepDetector/compact/materials.xml

```diff
@@ -172,5 +172,10 @@
     <D value="1.396" unit="g/cm3" />
     <composite n="1" ref="Ar"/>
   </material>
-
+  
+  <material name="Lead">
+    <D type="density" unit="g/cm3" value="11.35"/>
```

Comment:
I know it's very picky but could you please change the order of `unit` and `value` such that it's consistent with the others?

Reply:
If you look at it 50% use this ordering and the other 50% use value - unit ...
I will modify the entire file so that they all have type - value - unit (which seems more natural)

Reply:
I just looked at the lines before but the natural look was also my motivation ;) thank you :)